ftdc/cmd/parser.go

```diff
@@ -36,6 +39,31 @@ type gnuplotWriter struct {
 	options graphOptions
 }
 
+type kvPair[K, V any] struct {
+	Key K
+	Val V
+}
+
+func sorted[K cmp.Ordered, V any](mp map[K]V) []kvPair[K, V] {
```

Comment:
Hah love it. Normally I've seen
1. Build a slice of the keys
2. `sort.Strings` that slice 
3. Iterate through sorted slice to access map in key order

But this definitely also works.